src/xmpp/roster_list.c

```diff
@@ -199,6 +199,12 @@ roster_get_msg_display_name(const char* const barejid, const char* const resourc
 {
     assert(roster != NULL);
 
+    char* incoming_str = prefs_get_string(PREF_INCOMING_STR);
```

Comment:
what is this part doing?

Reply:
returns the value of "/stamp incoming" string.